cmd/eval.go

```diff
@@ -78,15 +87,22 @@ Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
 				combination.WithSolveAhead(),
 			)
 
+			generator, err := generate.NewGenerator(file, combinations)
```

Comment:
suggestion: Maybe we should either rename the `generate` package, the `Generator` struct (or both)? As it is, it doen't tell us much about _what_ is being generated.

Reply:
What would you think about renaming each to be `template` and `Builder` respectively? So it would end up looking like this:

```go
import (
    "fmt"
    "github.com/operator-framework/combo/pkg/template"
)

// (...)

templateBuilder, err := template.NewBuilder(templateFile, combinations)
if err != nil {
    return fmt.Errorf("failed to construct generator: %w", err)
}

ctx, cancel := context.WithCancel(context.Background())
defer cancel()

combinedTemplateDocuments, err := templateBuilder.Build(ctx)
if err != nil {
	return fmt.Errorf("failed to generate documents with combinations: %w", err)
}
```